e2e/src/deploy/synapse/index.ts

```diff
@@ -93,3 +94,15 @@ export async function stopSynapse() {
     console.log('Stopped synapse');
   }
 }
+
+function getIpAddress(container: StartedTestContainer): string {
```

Comment:
Same as we already use here https://github.com/nordeck/element-web-modules/blob/main/e2e/src/deploy/synapse/index.ts#L150